### PR TITLE
Move derived stats to v2. Add v2 container spec.

### PR DESCRIPTION
**Pull request by rjnagal**

### CODE REVIEW

**Comment by rjnagal** [comment]
@vmarmol Let me know what you think about adding conversion layer vs changing the internal structs.

File: info/v2/container.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+import (
+	"time"
+)
+
+type CpuSpec struct {
+	// Requested cpu shares. Default is 1024.
+	Limit uint64 `json:"limit"`
+	// Requested cpu hard limit. Default is unlimited (0)
+	MaxLimit uint64 `json:"max_limit"`
```
Comment (vmarmol) [inline info/v2/container.go]: Can we add units?

File: info/v2/container.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	// Cpu affinity mask.
+	Mask string `json:"mask,omitempty"`
```
Comment (vmarmol) [inline info/v2/container.go]: Maybe add a TODO to make this a set of sorts? WDYT?

**Comment by vmarmol** [comment]
If we go with a different internal struct we have future flexibility but conversions to v1 and v2. If we go with v2 we only need to convert to v1 which should go away over time. I tend to favor the second since I'm not sure how much more different we could make an internal struct. WDYT?

File: info/v2/container.go (hunk 2 above)
Comment (rjnagal) [inline info/v2/container.go]: I feel like string is more compact and manageable.  Its easier for people looking for visual input and straightforward for advanced users to convert.

File: info/v2/container.go (hunk 1 above)
Comment (rjnagal) [inline info/v2/container.go]: done. Looks like we don't fill it in yet.

**Comment by rjnagal** [comment]
Ok. I think I am going to build v2 with v1 conversion and slowly wean off internal usage from v1 to v2.

File: info/v2/container.go (hunk 2 above)
Comment (vmarmol) [inline info/v2/container.go]: SGTM, can we add a TODO to write a small library to do this? :) We already have it, we just need to export it here.

**Comment by vmarmol** [comment]
Plan SGTM

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**